### DISCUSSION TITLE
bidi interface in selenium hub

### DISCUSSION THREAD

**Discussion by andreRibeiro1989** [SeleniumHQ/docker-selenium#2395] [General]
how can I use bidi with the remote driver? This would work fine with a local driver
```
browser_options.enable_bidi = True
log_entries = []
driver.script.add_console_message_handler(log_entries.append)
```

**Comment by VietND96** [comment]
You need to set in the request, it would be similar to https://github.com/SeleniumHQ/docker-selenium/issues/2220
